test/integration/issue-589-constraint.bats

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+	swarm_manage_cleanup
+	stop_docker
+}
+
+@test "issue 589 node constraint regexp should support pipe" {
```

Comment:
Could you remove the reference to the issue in the file and the test and put everything into `constraints.bats`? I think the test description is explicit enough and it belongs here.